src/amo/pages/Home/index.js

```diff
@@ -194,28 +137,24 @@ export class HomeBase extends React.Component {
   }
 
   renderHeroHeader() {
```

Comment:
I made a mistake with this thing. This is included on mobile, which means that we need `homeShelves` on mobile as well as on desktop. This will change only the saga, I think, but we need to make sure that we call the API and load the data from the shelves API, even on mobile.